rke/k8s_rke_system_images.go

```diff
@@ -1977,12 +1977,12 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  m("rancher/kubelet-pause:v0.1.3"),
 			Nodelocal:                 m("k8s.gcr.io/k8s-dns-node-cache:1.15.7"),
 		},
-		// Enabled in v2.4.4
+		// Enabled out of band after v2.4.3
 		// Reminder: Save template rancher1-1 for k8s 1.15 for Rancher v2.2.x due to WindowsPodInfraContainer image
 		// Reminder: This template contains Nodelocal image which isn't in templates before v2.3.7
-		"v1.15.12-rancher1-2": {
+		"v1.15.12-rancher2-2": {
```

Comment:
We'd need to save rancher2-2 for rancher 2.2?

Reply:
Technically, no, but I figured I might as well save it. :) 

The `rancher2` images are all the same image but has the Windows 1909 as part of the manifest.